### PR TITLE
#265 refactor: package cleanup

### PR DESCRIPTION
**Pull request by macintoshpie**
#### Any background context you want to provide?
GMT has moved to a different approach for modeling and connecting modelica components.

For future reference, the tag here can be used for seeing the tool before this large purge of files:
https://github.com/urbanopt/geojson-modelica-translator/releases/tag/old-design

#### What does this PR accomplish?
Clean up files from old GMT approach:
- remove old python files
- remove dangling modelica files
- fix file names (remove `_new` and `New`)

#### How should this be manually tested?
If tests were passing, run one of the tests and load it into dymola.

#### What are the relevant tickets?
#265

### CODE REVIEW

File: geojson_modelica_translator/model_connectors/model_base.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def __init__(self, system_parameters):
         self.system_parameters = system_parameters
 
         # initialize the templating framework (Jinja2)
-        self.template_dir = Path(__file__).parent / "templates"
+        self.template_dir = template_dir
```
Comment (macintoshpie) [inline geojson_modelica_translator/model_connectors/model_base.py:57]: templates are now separated by the model type directory (ie load, ets, network)

**Review by nllong — COMMENTED** [review]
Model ran successfully in Dymola. There seem to be some instabilities in the data, though. I'll leave that for another day to solve.

image

File: geojson_modelica_translator/geojson_modelica_translator.py [hunk 2]
```diff
@@ -115,8 +114,7 @@ def process_loads(self):
                 raise SystemExit(f'Model of type {model_con} not recognized. Verify sysparam file')
```
Comment (nllong) [inline geojson_modelica_translator/geojson_modelica_translator.py:114]: I like the use of `SystemExit`. I typicaly just raise Exception... but this looks cleaner.

File: geojson_modelica_translator/model_connectors/energy_transfer_systems/cooling_indirect.py [hunk 3]
```diff
@@ -50,7 +50,7 @@ def to_modelica(self, scaffold):
 
         :param scaffold: Scaffold object, Scaffold of the entire directory of the project.
         """
-        cooling_indirect_template = self.template_env.get_template("CoolingIndirectNew.mot")
+        cooling_indirect_template = self.template_env.get_template("CoolingIndirect.mot")
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/energy_transfer_systems/cooling_indirect.py:53]: 👍  This is going to be an exciting cleanup!

Comment (nllong) [inline geojson_modelica_translator/model_connectors/energy_transfer_systems/cooling_indirect.py:53]: 👍  This is going to be an exciting cleanup!

File: geojson_modelica_translator/model_connectors/energy_transfer_systems/energy_transfer_base.py [hunk 4]
```diff
@@ -35,3 +37,6 @@ class EnergyTransferBase(ModelBase):
     Base class of the energy transfer connectors.
     """
     simple_gmt_type = 'ets'
+
+    def __init__(self, system_parameters):
+        super().__init__(system_parameters, Path(__file__).parent / 'templates')
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/energy_transfer_systems/energy_transfer_base.py:42]: nice... you are following the use of Path from @vtnate 😛

File: geojson_modelica_translator/model_connectors/load_connectors/teaser.py [hunk 5]
```diff
@@ -136,14 +142,8 @@ def to_modelica(self, scaffold, keep_original_models=False):
         finally:
             os.chdir(curdir)
 
-        # Process each building
-        # TODO: break this out?
-
         self.post_process(scaffold, building_names, keep_original_models=keep_original_models)
 
-    def process_building(self, scaffold, building_name):
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/load_connectors/teaser.py:144]: clean up, clean up, everybody clean up

File: tests/management/test_format_modelica_files.py [hunk 6]
```diff
@@ -50,18 +51,17 @@ def setUp(self):
...
     def test_no_meaningful_diff_when_formatting_mot_files(self):
         """After applying formatter to .mot (Jinja) files, we expect the only differences to be in whitespace"""
-        for file_ in os.listdir(self.template_dir):
-            filepath = os.path.join(self.template_dir, file_)
-            outfilepath = os.path.join(self.output_dir, file_)
+        for file_ in TEMPLATE_FILES:
+            outfilepath = os.path.join(self.output_dir, file_.name)
 
-            if not file_.endswith(".mot") or os.path.basename(file_) in SKIP_FILES:
+            if file_.suffix != ".mot" or file_.name in SKIP_FILES:
```
Comment (nllong) [inline tests/management/test_format_modelica_files.py:57]: I like the use of SKIP_FILES.

**Review by nllong — APPROVED** [review]
This looks great. There are a lot of files in the PR, but most of them have been moved or deleted (rightly so). This cleanup is going to be much more maintainable!

File: geojson_modelica_translator/model_connectors/energy_transfer_systems/energy_transfer_base.py (hunk 4 above)
Comment (macintoshpie) [reply geojson_modelica_translator/model_connectors/energy_transfer_systems/energy_transfer_base.py:42]: yup! I didn't even know about the package until I saw him use it

**Pull request merged**